R/murphy_diagram.R

```diff
@@ -0,0 +1,80 @@
+#' Murphy diagram
+#'
+#' Murphy diagram for elementary scores. Can be used to compare performance of one or multiple models across ranges of thetas for elementary scores. If the plot needs to be customized, set \code{plot = FALSE} to get the resulting data instead of the plot.
+#'
+#' @importFrom graphics matplot legend
+#' @param actual Observed values.
+#' @param predicted Predicted values.
+#' @param w Optional case weights.
+#' @param alpha Level of expectile resp. quantile.
```

Comment:
Maybe mention again that the default `alpha=0.5` corresponds to the expectation resp. median.

Reply:
Added, thx.